pkg/blackduck/ctl_blackduck.go

```diff
@@ -218,7 +218,8 @@ func (ctl *Ctl) AddSpecFlags(cmd *cobra.Command, master bool) {
 	cmd.Flags().StringVar(&ctl.ProxyCertificateFilePath, "proxy-certificate-file-path", ctl.ProxyCertificateFilePath, "Absolute path to a file for the Black Duck proxy certificate")
 	cmd.Flags().StringVar(&ctl.AuthCustomCAFilePath, "auth-custom-ca-file-path", ctl.AuthCustomCAFilePath, "Absolute path to a file for the Custom Auth CA for Black Duck")
 	cmd.Flags().StringVar(&ctl.Type, "type", ctl.Type, "Type of Blackduck")
-	cmd.Flags().StringVar(&ctl.DesiredState, "desired-state", ctl.DesiredState, "Desired state of Blackduck")
+	cmd.Flags().BoolVar(&ctl.Start, "start", ctl.Start, "Start the Blackduck")
```

Comment:
@rrati whether there should be any default behavior?

Reply:
No.  If no options are given we don't want to change anything, nor is there a good default value since we don't want to do anything if neither option is set.